exasol/ds/sandbox/lib/dss_docker/create_image.py

```diff
@@ -162,10 +168,10 @@ def _commit_container(
         notebook_folder_initial = get_fact(facts, "notebook_folder", "initial")
         conf = {
             "Entrypoint": entrypoint(facts),
-            "User": get_fact(facts, "jupyter", "user"),
             "Cmd": [],
             "Volumes": {notebook_folder_final: {}, },
             "ExposedPorts": {f"{port}/tcp": {}},
+            "User": get_fact(facts, "docker_runner"),
```

Comment:
docker_runner creates the wrong associations, I think.

Maybe docker_user or docker_default_user are more clear